simple_rpc/simple_rpc.py

```diff
@@ -120,22 +143,19 @@ def open(self):
 
     def close(self):
         """Disconnect from device."""
-        if not self.is_open():
-            return
-
         for method in self.methods:
             delattr(self, method)
+        self.methods.clear()
 
-        self.methods = {}
-
-        if (self._connection):
-            self._connection.close()
+        self._close()
 
     def is_open(self):
-        """Query device state."""
+        """Query interface state."""
```

Comment:
* This method always returned false when called by the user when a socket is used. This patch fixes that.